beginner_source/translation_transformer.py

```diff
@@ -0,0 +1,399 @@
+"""
+Language Translation with Transformer
+=====================================
+**Author**: `Jayasimha T <https://github.com/iamsimha>`_
```

Comment:
For the NLP domain, in general, we don't include author's name in the new tutorials. You contribution is tracked automatically.